### DISCUSSION TITLE
Radar chart and sum

### DISCUSSION THREAD

**Discussion by EmanuelFromFlorence** [chrispahm/chartjs-plugin-dragdata#76] [Ideas]
Hi , 
don't know if this is the right place to ask for but
I saw the radar chart and I was wondering , is it possiible to show the total amount of votes ( of all the areas -axis's?)
and is is possible to send this data to a form ?
I'm really new at Javascript but I have a task where this would be really useful.
Can anybody show this to me here?
By the way this is so great so thanks.

**Comment by chrispahm** [comment]
Hey @EmanuelFromFlorence,

sorry for taking me so long! What you want to achieve is fairly simple and straightforward in Javascript (and using this plugin 😎).
Here's a small fiddle with comments:
https://jsfiddle.net/3f8Laowj/1/

**Comment by EmanuelFromFlorence** [comment]
Wow Christopher this is so kind from you ,
thanks and i understand you have a lot of things
to do so thanks for your time,
i like so much this plug in so thanks for it
and...thanks for the chance to learn in
such a beautiful way...

stay safe.
cheers from Florence - Italy
ps.
if you need someone to translate in italian
feel free to aks it to me...
it would be a real pleasure
ᐧ

**Reply by EmanuelFromFlorence** [reply]
Ps.Christopher ...in this you write the method to send the results to a database ?
 // send to database
              /*
              await fetch('https://my-api.org/votes', {
                method: 'POST',
                body: {
                  voteCount: newTotalVotes
                }
              })
              */
maybe its already clear but does this mean that i 
can send it to Airtable or a Google Sheet or is strictly to local ?
thanks

**Reply by chrispahm** [reply]
No this is exactly the idea, you can send it to airtable for instance using their API https://airtable.com/api

**Reply by EmanuelFromFlorence** [reply]
Didn't understand at first now i am, so i do apologize for bothering you,
chart.js , your kindness and help were and are really helpful ,now i'm stuck in maybe simpler thing , 
i cannot give to the radar chart the colors i want , for the fill of the area, for the grid lines , for the background and for the axis..
would you be so gentle to help me understanding ? I read the docs but i cannot find the clue , 
https://www.chartjs.org/docs/latest/charts/area.html
it says this but , while its clearly my own fault, no matter what i cannot apply this example

datasets: [
            {
              fill: {
                target: 'origin', (' # of votes ' in this case? )
                above: 'rgb(255, 0, 0)',   // Area will be red above the origin
                below: 'rgb(0, 0, 255)'    // And blue below the origin
              }
            }
        ]

also and last how can i format ,place more close or far or in the center the vote counter with the total number? On mobile is going outside and on some mobiles is never being shown... .Thanks for all...if you  can .

**Reply by chrispahm** [reply]
Hey Emanuel, thanks for reaching out!
Could you provide me with a minimum working example? E.g. on jsFiddle?

Best
Christoph

**Reply by EmanuelFromFlorence** [reply]
Sure, here it is. 
this is the one you so gently permit to study 
https://jsfiddle.net/EmanueleMarea/sad9q76u/19/#&togetherjs=sm4rK5zo72
but first of all ,thanks for replyn me and to give me 
a little of your time , so much appreciated
also you can see i tried , to send data by a click button
just the button then i got to ..take care of myself,
luckily i'm fine but just to explain what i was trying to do ;)
is that i cannot be able to put the elements where i want and
on mobile well its not working leaving out the total counter
and the lavel being so distant from graph..is clearly not a good UI
AND it all my fault...so thanks
and good morning if this is the case 
its mine from Florence ,Italy

thank you thank you thank you soo much

**Reply by EmanuelFromFlorence** [reply]
and i woud like to change colors of the Area ,grids, to be a little bit in control of the UI, i tried to edit with a css viewer , yellowpencil extension to understand which css rules i could apply to any id but..and i know for you its clear..no luck and all stretches and blurs, by the way thanks for dedicate a little of your time to me even just for reading till here or replyin me before

**Comment by EmanuelFromFlorence** [comment]


**Reply by EmanuelFromFlorence** [reply]
maybe this wasn't a pertinent question and this wasn't related to drag and drop so i do apologize but thanks for replyin to me

**Reply by chrispahm** [reply]
Hey no worries, I'll be checking your fiddle when I have a little more time at hand 😊

**Comment by EmanuelFromFlorence** [comment]
I was worried ..don’t wanna disturb you at all , so thanks , no rush for me
and thanks for your kindness,in the meanwhile i’ll try to study chart.js
better , have a good day

Il giorno gio 29 set 2022 alle 10:31 Christoph Pahmeyer <
***@***.***> ha scritto:
